src/main/scala/co/topl/brambl/builders/BuilderError.scala

```diff
@@ -1,3 +1,7 @@
 package co.topl.brambl.builders
 
 trait BuilderError
+
+object BuilderErrors {
+  case class InputBuilderError(message: String) extends BuilderError
```

Comment:
If BuildError is an abstract class then 
```suggestion
  case class InputBuilderError(message: String) extends BuilderError(message)
```